### ISSUE TITLE
Browser stalls during development saves

### ISSUE THREAD

**Issue by felixunivers** [tomastrajan/angular-ngrx-material-starter#455]
## I would be willing to submit a PR to fix this issue:

[ ] Yes (Assistance is provided if you need help submitting a pull request)  
[ ] No

**Comment by felixunivers** [comment]
Thank you for the starter - very good project for learning Angular techniques.
While experimenting with my own components I occasionally omit a bug or a typo and that gets the entire application into stall.  In browser debugger the count of errors goes to thousands and the browser stalls. The only way to get out of it is to kill the process. I am assuming this may have something to do with the Redux library. Is there a way to bypass it for development or instruct to simply stop on first error?

**Comment by timdeschryver** [comment]
Hi, do you got a reproduction of this? This shouldn't be happening... 🤔

**Comment by felixunivers** [comment]
Sorry, for the dealy. I tried now some errors and compiling on the original 'starter' project and I do not seam to be able to reproduce the behavior there. However, in my project based on the 'starter' where I have made some additions (not too many), I am experiencing the described issues. I tried to isolate it, but my experience with Angular and TS is still rather of a "semi-educated novice" and I could not get to the bottom of it. If someone is eager to dig around , I'll be happy to upload my project somewhere.
